lib/galaxy/managers/collections.py

```diff
@@ -46,11 +46,11 @@ def __init__(self, app):
         self.tag_manager = tags.GalaxyTagManager(app.model.context)
         self.ldda_manager = lddas.LDDAManager(app)
 
-    def precreate_dataset_collection_instance(self, trans, parent, name, structure, implicit_inputs=None, implicit_output_name=None):
+    def precreate_dataset_collection_instance(self, trans, parent, name, structure, implicit_inputs=None, implicit_output_name=None, tags=[]):
```

Comment:
I guess we'll want to avoid tags being mutable here.

Reply:
Opps... definitely. Thanks for the catch.